packages/rsx_interpreter/src/lib.rs

```diff
@@ -83,14 +84,12 @@ macro_rules! get_line_num {
     () => {{
         let line = line!();
         let column = column!();
-        let file = file!();
+        let file_path = file!().to_string();
+        let mut crate_path = env!("CARGO_MANIFEST_DIR").to_string();
```

Comment:
Now that they're separate things, we don't require any formatting. So I think we can avoid allocating and just store them as `&'static str`? (blazing fast 🚀🚀)

Also the `mut` seems unnecessary

Reply:
This seemed like a good idea, but I don't think it works. The macro will produce a &'static str, and we can get serde to work with it, but the cli cannot produce &'static str when it detects changes in the code, so I'm going to revert this change.

Reply:
Ah true, I didn't know the CLI constructs it as well